File: lib/widgets/stats/kpi_stat_card.dart
import 'package:flutter/material.dart';
import 'stat_card.dart';

/// @deprecated Use StatCard with style: StatCardStyle.kpi instead
class KpiStatCard extends StatelessWidget {
  final String title;
  final String value;
  final IconData icon;
  final Color color;

  const KpiStatCard({
    super.key,
    required this.title,
    required this.value,
    required this.icon,
    required this.color,
  });

  @override
  Widget build(BuildContext context) {
    return StatCard(title: title, value: value, icon: icon, color: color);
  }
}

File: app/src/main/java/io/github/benji377/timety/ui/components/stats/KpiStatCard.kt
package io.github.benji377.timety.ui.components.stats

import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.graphics.vector.ImageVector

/**
 * @deprecated Use StatCard with style: StatCardStyle.KPI instead
 */
@Deprecated("Use StatCard with style = StatCardStyle.KPI instead")
@Composable
fun KpiStatCard(
    title: String,
    value: String,
    icon: ImageVector,
    color: Color,
    modifier: Modifier = Modifier
) {
    StatCard(
        title = title,
        value = value,
        icon = icon,
        color = color,
        modifier = modifier,
        style = StatCardStyle.KPI
    )
}
